### PR TITLE
server: sftpgo

### PR DESCRIPTION
**Pull request by alichaddad**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,32 @@
+FROM docker.io/golang:alpine as builder
```
Comment (brandonpille) [inline Dockerfile:1]: Why do we need this?

Comment (alichaddad) [reply Dockerfile:1]: That's so we can make the deployed image more lightweight without extra dependencies

**Review by DylanVerstraete — CHANGES_REQUESTED** [review]
Looks good, one comment

File: Dockerfile (hunk 1 above)
Comment (brandonpille) [reply Dockerfile:1]: No I meant why do we need this dockerfile?

Comment (alichaddad) [reply Dockerfile:1]: We need a docker compose setup to run web3 proxy with document server as requested in this [ticket](https://github.com/freeflowuniverse/aydo/issues/2).

**Comment by DylanVerstraete** [comment]
@alichaddad please resolve conflicts

**Comment by xmonader** [comment]
Have to merge this now, for further improvements let's create more tickets if needed

**Comment by brandonpille** [comment]
> Have to merge this now, for further improvements let's create more tickets if needed

I can understand that there is a rush to merge PRs but we should still properly finish a review before merging. Just merging things without the review being completely done is how we introduce bugs. It's also how we introduce code in places where it doesn't belong. For example I don't think this docker compose file belongs here.

Please don't just merge PRs but instead reach out to the reviewer directly if it's urgent.

**Pull request merged**